src/quickstart/_main.mdx

```diff
@@ -0,0 +1,34 @@
+<!-- prettier-ignore-start  -->
+<script>
+  /* import all the Airnote quickstart mdx files into the _main mdx file to make them available. See mdsvex example here: https://mdsvex.pngwn.io/playground*/ 
+  import Introduction from '../quickstart/01-introduction.mdx'; 
+  import Unboxing from '../quickstart/02-unboxing-your-airnote.mdx'; 
+  import Setup from '../quickstart/03-airnote-setup.mdx';
+  import Dashboard from '../quickstart/04-visit-your-device-dashboard.mdx'; 
+  import AirQuality from '../quickstart/05-understanding-air-quality.mdx'; 
+  import FAQ from '../quickstart/06-faq.mdx';
+</script>
+<!-- prettier-ignore-end -->
+
+# Airnote Quickstart
```

Comment:
So this is an `h1` now, but with the current CSS `h1` and `h2` elements are both 24 pixels.

Reply:
On blues.dev `h1` elements are 32 pixels so I’d mirror that.

Reply:
Done